clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/processor/JustificationProcessor.java

```diff
@@ -121,6 +126,20 @@ public AuditLog.Builder auditLogBuilderWithJustification(
           Value.newBuilder().setStructValue(justificationStruct).build());
       auditLogBuilderCopy.setMetadata(metadataBuilder.build());
 
+      Claim justificationsClaim = jwt.getClaim("justs");
+      if (justificationsClaim.isNull()) {
+        log.warn("can't find 'justs' in claims");
+      } else {
+        RequestMetadata.Builder requestMetadataBuilder =
+            auditLogBuilderCopy.getRequestMetadataBuilder();
```

Comment:
I saw that line 123 uses `auditLogBuilderCopy.getMetadata().toBuilder()`. Does requestMetadata has the same function `auditLogBuilderCopy.getRequestMetadat().toBuilder()`? If there is, what's the difference?

Reply:
GetXXBuilder() or getXX().toBuilder() will return a default/empty instance if it is null. The different between the two method is one returns a Builder one returns an object, use getXXBuilder() since it is all we need otherwise will need to convert the object to builder to set values in it.

The logic is what you mentioned, it only adds the reason and does not alter anything else when justs is found and non0null/empty.

Reply:
ack. Thank you for the explanation and confirmation!